.changeset/cold-deer-cheat.md

```diff
@@ -0,0 +1,20 @@
+---
+"@fluidframework/container-definitions": major
+"@fluidframework/container-loader": major
+"@fluidframework/container-runtime": major
+"@fluidframework/datastore": major
+"@fluidframework/driver-definitions": major
+"@fluidframework/driver-utils": major
+"fluid-framework": major
+"@fluidframework/odsp-doclib-utils": major
+"@fluidframework/odsp-driver": major
+"@fluidframework/odsp-driver-definitions": major
+"@fluidframework/routerlicious-driver": major
+"@fluidframework/telemetry-utils": major
+"@fluid-private/test-end-to-end-tests": major
+"@fluidframework/tinylicious-client": major
```

Comment:
Please change all of these to "minor" instead of major. This is just a side effect because the change originated on the next branch.

Reply:
Changed to `minor`.